packages/manager/modules/account/src/user/dashboard/user-dashboard.constants.js

```diff
@@ -48,22 +63,36 @@ export const USER_DASHBOARD_SHORTCUTS = [
       path: '#/billing/autorenew',
     },
     regions: ['EU', 'CA'],
+    tracking_name_suffix: 'manage-my-services',
   },
   {
     key: 'MANAGE_USERS',
     url: {
       baseURL: 'iam',
       path: '/dashboard/users',
     },
-    new: true,
+    tracking_name_suffix: 'manage-my-users',
   },
   {
     key: 'ADD_CONTACT',
     state: 'account.contacts.services',
     regions: ['EU'],
+    tracking_name_suffix: 'add-contact',
   },
 ];
 
+const TRACKING_CHAPTER_1 = 'hub';
```

Comment:
Chapter 1 should be capitalized 
```suggestion
const TRACKING_CHAPTER_1 = 'Hub';
```